fennel/test_lib/mock_client.py

```diff
@@ -72,6 +71,61 @@ def __init__(self, status_code: int, content: str):
         )
 
 
+def _dataframe_lookup(
+    dataset_name: str,
+    keys: pd.DataFrame,
+    right_df: pd.DataFrame,
+    join_columns: List[str],
+    timestamp_field: str,
+) -> pd.DataFrame:
```

Comment:
Can you add a comment / description to this function

Reply:
yup added.